test/mocks/server/admin_stream.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <chrono>
```

Comment:
Are we scripting the removal of extra #includes or doing it manually? We don't need most of these system headers in the new headers.

Reply:
I remove them manually, just forget to remove them. Thanks for pointing out